packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1370,6 +1373,22 @@ export default mixins(
 				this.createNodeActive = false;
 			},
 
+			addStickyNote() {
+				if (document.activeElement) {
+					(document.activeElement as HTMLElement).blur();
+				}
+
+				const offset: [number, number] = [...(this.$store.getters.getNodeViewOffsetPosition as [number, number])];
+
+				const position = CanvasHelpers.getMidCanvasPosition(this.nodeViewScale, offset);
+				position[0] -= DEFAULT_STICKY_WIDTH / 2;
+				position[1] -= DEFAULT_STICKY_HEIGHT / 2;
```

Comment:
New stickies are not getting added to the center for me. You probably have to add the `SIDEBAR_WIDTH` and `HEADER_HEIGHT` into the offset calculation here somewhere (?).

See screenshot below, the position with respect to the execute button.

<img width="3200" alt="image" src="https://user-images.githubusercontent.com/6179477/180433979-68a3a08a-777a-44fe-a855-dbd09e8dd306.png">

Reply:
I don't think it has to be perfect. will double check with David

Reply:
checked with David. this is good. sidebar width is accounted for actually

Reply:
it's weird. but not worth the effort